sqlx-core/src/postgres/types/mod.rs

```diff
@@ -210,6 +211,7 @@ mod bit_vec;
 
 pub use array::PgHasArrayType;
 pub use interval::PgInterval;
+pub use ltree::PgLTree;
```

Comment:
This means `ltree::Error` isn't publicly accessible. If you want it to be, it should be re-exported here, but `Error` is too generic of a name for this context. `PgLTreeParseError` maybe?